### DISCUSSION TITLE
Can't run my code

### DISCUSSION THREAD

**Discussion by Yulongggggg** [idaholab/moose#28630] [Q&A Modules: Solid mechanics] [answered]
This is my code, I have tried multiply times but still can't make it run
![Screenshot 2024-09-16 at 11 38 58 AM](https://github.com/user-attachments/assets/4f73a491-2812-411f-9e80-348fc6a5b9cc)

**Comment by GiudGiud** [comment]
hello

Please attach a file instead of a screenshot.
And explain your problem you are running into in more details

Guillaume

**Reply by Yulongggggg** [reply]
[code.zip](https://github.com/user-attachments/files/17016349/code.zip)
[code.zip](https://github.com/user-attachments/files/17016349/code.zip)
I can get the strain and stress data in the paraview, but I think I have used the command 'generate_output' to get the stress and strain. Could you please tell me my mistakes, I really appreciate it.

**Reply by Yulongggggg** [reply]
sorry, I can't get strain and stress

**Reply by GiudGiud** [reply]
Could you please upload the inputs to a public github repo? We cant download zip files on our machines
generate_output goes inside the Physics, under QuasiStatic/all

**Reply by Yulongggggg** [reply]
Of course! This is my link:https://github.com/Yulongggggg/1D_Bar_MOOSE .Thank you Guillaume!

**Reply by GiudGiud** [reply, marked as answer]
There should not be a comma inside generate_output
you can try this
```
 generate_output = 'max_principal_stress min_principal_stress stress_xx stress_xy stress_xz stress_yy stress_yz stress_zz'
```

**Reply by Yulongggggg** [reply]
Hi Guillaume, problem solved! Thank you so much! Btw, I have a question, if I have settled strain and strain in the 'generate_output', do I still have to define the shape function, like 
```
# [AuxVariables]
#   [stress_theta]
#     order = CONSTANT
#     family = MONOMIAL
#   []
#   [strain_theta]
#     order = CONSTANT
#     family = MONOMIAL
#   []
# []
```

**Reply by GiudGiud** [reply]
you dont need to define the order of the auxvariables, the automated material property output does it

**Reply by Yulongggggg** [reply]
Thank you so much, love you guy
